app/controllers/appeal-application/fee-support.ts

```diff
@@ -0,0 +1,59 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import _ from 'lodash';
+import { FEATURE_FLAGS } from '../../data/constants';
+import { paths } from '../../paths';
+import LaunchDarklyService from '../../service/launchDarkly-service';
+import UpdateAppealService from '../../service/update-appeal-service';
+
+function getFeeSupportQuestion(appeal: Appeal) {
+  let hint: string;
+  let decision: string;
+  hint = '';
+  const question = {
+    title: 'title',
+    hint,
+    options: [
+      {
+        value: '',
+        text: '',
+        checked: decision === ''
+      },
+      {
+        value: '',
+        text: '',
+        checked: decision === ''
+      }
+    ],
+    inline: false
+  };
+  return question;
+}
+
+async function getFeeSupport(req: Request, res: Response, next: NextFunction) {
+  try {
+    const paymentsFlag = await LaunchDarklyService.getInstance().getVariation(req, FEATURE_FLAGS.CARD_PAYMENTS, false);
+    if (!paymentsFlag) return res.redirect(paths.common.overview);
```

Comment:
Do we need a flag   

 ` const drlmSetAsideFlag = await LaunchDarklyService.getInstance().getVariation(req, FEATURE_FLAGS.DLRM_SETASIDE_AIP_FEATURE_FLAG, false);`

The one I added in decision-type?

Reply:
This controller can be deleted after you make your changes. This is just to make the link works.